tests/containers/docker_firewall.pm

```diff
@@ -61,7 +62,7 @@ sub run {
     script_retry "docker ps -q | wc -l | grep 0", delay => 5, retry => 12;
 
     # Test the connectivity of Docker containers
-    $engine->check_containers_connectivity();
+    check_containers_connectivity($engine);
```

Comment:
```suggestion
    check_containers_connectivity($engine->runtime);
```
maybe?

Reply:
It's not really necessary. Please see #13408

Reply:
Yes, sorry, I forgot about that magic :)